todo-demo-app/src/test/resources/todo-demo-app.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
+  metadata:
+    name: todo-demo-app
+  spec:
+    lookupPolicy:
+      local: false
+
+- apiVersion: build.openshift.io/v1
+  kind: BuildConfig
+  metadata:
+    name: todo-demo-app
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: todo-demo-app:latest
+    source:
+      git:
+        uri: https://github.com/quarkusio/todo-demo-app.git
+      type: Git
+    strategy:
+      sourceStrategy:
+        env:
+          - name: ARTIFACT_COPY_ARGS
+            value: -p -r lib/ *-runner.jar
+        from:
+          kind: ImageStreamTag
+          name: openjdk-11-rhel7:latest
+      type: Source
+    triggers:
+    - type: ConfigChange
+    - imageChange: {}
+      type: ImageChange
+
+- apiVersion: apps.openshift.io/v1
+  kind: DeploymentConfig
+  metadata:
+    name: todo-demo-app
+  spec:
+    replicas: 1
+    selector:
+      name: todo-demo-app
+    template:
+      metadata:
+        labels:
+          name: todo-demo-app
+      spec:
+        containers:
+        - env:
+          - name: ARTIFACT_COPY_ARGS
+            value: -p -r lib/ *-runner.jar
+          image: todo-demo-app:latest
+          name: todo-demo-app
+          ports:
+          - containerPort: 8080
+            protocol: TCP
+    test: false
+    triggers:
+    - type: ConfigChange
+    - imageChangeParams:
+        automatic: true
+        containerNames:
+        - todo-demo-app
+        from:
+          kind: ImageStreamTag
+          name: todo-demo-app:latest
+      type: ImageChange
+
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: todo-demo-app
+  spec:
+    ports:
+    - name: 8080-tcp
+      port: 8080
+      protocol: TCP
+      targetPort: 8080
+    selector:
+      deploymentconfig: todo-demo-app
```

Comment:
I wonder how this YAML can work, given that no resource has this `deploymentconfig` label :-)